### DISCUSSION TITLE
SUS DEACTIVATED

### DISCUSSION THREAD

**Discussion by ItsEnderium** [CluckCluckChicken/Magnifier#92] [General]
[YO MAGNIFIER IN SCRATCH ADDONS POGGERS](https://github.com/ScratchAddons/ScratchAddons/discussions/2675)

**Comment by JackyBorderCollie** [comment]
yeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

**Comment by JackyBorderCollie** [comment]
add epik gamer moment labelllllllll plls
